hazelcast/src/main/java/com/hazelcast/jet/impl/deployment/ChildFirstClassLoader.java

```diff
@@ -33,11 +39,23 @@
  */
 public class ChildFirstClassLoader extends URLClassLoader {
 
+    private static final ILogger LOG = Logger.getLogger(ChildFirstClassLoader.class);
+
     private volatile boolean closed;
 
     public ChildFirstClassLoader(@Nonnull URL[] urls, @Nonnull ClassLoader parent) {
         super(urls, parent);
 
+        for (URL url : urls) {
+            try {
+                if (!new File(url.toURI()).exists()) {
+                    LOG.warning("URL " + url + " does not exist.");
```

Comment:
The URL can be something else than file:
```suggestion
                    LOG.warning("URL '" + url + "' does not point to an existing local file.");
```